remove_test.go

```diff
@@ -0,0 +1,100 @@
+//Copyright 2013 GoGraphviz Authors
```

Comment:
2018 and you are are welcome to add yourself to the authors or contributors as you see appropriate

Reply:
Will do